src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -13,40 +13,46 @@ import './DropdownMenu.scss';
 import classNames from 'classnames';
 
 export interface DropdownItem {
+  /** Unique ID to use in Array.map() */
   id: string;
+  /** Label to appear in the menu */
   label: string | ReactNode;
+  /** Callback to run when an item is selected */
```

Comment:
do we need these comments? they don't really seem to provide much benefit.

Reply:
The intention was to improve DX as per below:
![Screen Shot 2022-09-21 at 1 47 03 PM](https://user-images.githubusercontent.com/12437916/191395860-4ac3aca4-c2bb-48ac-a47d-4f2da56c3977.png)

However, looks like these comments aren't included in the build. That puts a damper on things!

Reply:
ah gotcha. .. :thinking: yeah. .. that makes sense. if you can figure out how make that work, i agree that it would be pretty useful in this type of situation.